x-pack/platform/plugins/shared/observability_ai_assistant/server/analytics/recall_ranking.ts

```diff
@@ -21,7 +21,7 @@ const schema: RootSchema<RecallRanking> = {
     type: 'array',
     items: {
       properties: {
-        elserScore: {
+        esScore: {
           type: 'float',
           _meta: {
             description: 'The score produced by ELSER text_expansion',
```

Comment:
nit: we should update the description as well:
```suggestion
        esScore: {
          type: 'float',
          _meta: {
            description: 'The score produced by Elasticsearch',
```

Btw. I am not sure if you need to handle mapping changes elsewhere too. Try asking `@afharo`

Reply:
Apologies for the late reply. I can see that this is an EBT server-side event.

In that case, if you are relying on the generic index `ebt-kibana-server` ([docs](https://docs.elastic.dev/telemetry/collection/kibana-general-purpose-indices#kibana-ebt-server)), no mapping update is required (maybe a runtime field update in your dataview).

If your team already has an indexer or a BigQuery pipeline leveraging this event type (`observability_ai_assistant_recall_ranking`), then you should update it accordingly. 

cc @elastic/platform-analytics

Reply:
Thanks @afharo